src/simulate/config/logging.ini

```diff
@@ -0,0 +1,27 @@
+[loggers]
```

Comment:
Although I wrote earlier for `props.ini` below that logging config seems to require `.ini` files, that's not the case.  You can replace `logging.config.fileConfig()` with a combination of `yaml.safe_load()` and `logging.config.dictConfig()` to read the config from a YAML file.